src/frontendHighlighterApp/index.js

```diff
@@ -356,7 +356,7 @@ class AccessibilityCheckerHighlight {
 						<button id="edac-highlight-next" disabled="true">Next<span aria-hidden="true"> »</span></button><br />
 					</div>
 					<div>
-						<button id="edac-highlight-disable-styles" class="edac-highlight-disable-styles" aria-live="polite">Disable Styles</button>
+						<button id="edac-highlight-disable-styles" class="edac-highlight-disable-styles" aria-live="polite" aria-label="Disable Page Styles">Disable Styles</button>
```

Comment:
I know there's other strings in the is file that aren't translated so this isn't a hard blocker but if it's easy for you to wrap the strings on this line in translation functions that'd be great.